ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -44,15 +47,22 @@ tests:
     make lint
   container:
     clone: true
-    from: golangci-lint-shellcheck
+    from: test-bin
+- as: helm-lint
+  commands: |
+    export XDG_CACHE_HOME=/tmp/.cache
+    make helm-lint
+  container:
+    from: test-bin
+  optional: true
 - as: unit
   commands: make unit-test
   container:
-    from: src
+    from: test-bin
```

Comment:
Why do we need `test-bin` for `make unit-test`? Can this remain `src`?

Reply:
I just though it will be easier to have a single container which contains all our dependencies and use it everywhere so we don't need to update the CI for each new dependency on SRO.

Reply:
But no such dependency is needed for this target, and `src` comes for free, right? So I would leave `src` here and change whenever we need it (and I hope we never do - unit tests should always remain pure Go).

Reply:
Moved back to src